test/chpldoc/include/DocsInclude.chpl

```diff
@@ -0,0 +1,6 @@
+/* docs string */
```

Comment:
Who's to say that this is a docs string comment?  Isn't any comment preceding an `include` going to be considered a potential docs comment?  (which suggests that only code may precede an `include` statement, which seems... unfortunate).  Maybe this error should only be triggered if the module itself doesn't have a comment but the include does?  Or maybe it should just be a warning in that case?  Or maybe we shouldn't worry about it at all and the user can read the documentation to learn that they should put chpldoc docs on module declarations not include statements?  I'm worried the current behavior will cause more false negatives than help.

Reply:
> Isn't any comment preceding an include going to be considered a potential docs comment?

No, a `//` comment wouldn't be.

> Maybe this error should only be triggered if the module itself doesn't have a comment but the include does?

I considered that but it also seems sortof fragile.

> Or maybe we shouldn't worry about it at all and the user can read the documentation to learn that they should put chpldoc docs on module declarations not include statements?

This is my current preference.

Reply:
OK, I agree.  So we'll remove this error checking case and related test?

Reply:
This is done.